### ISSUE TITLE
Grails 3.0.1 로 Upgrade.

### ISSUE THREAD

**Issue by herblover** [okjsp/okky#69]
Grails가 3.0.1이 나왔네요. ;;;

[딱 4일 됐습니다.](https://github.com/grails/grails-core/releases/tag/v3.0.1)

다른 부분보다, 다음과 같은 부분들은 okky 방문하는 개발자들이 관심을 가질 만한 특성이 있어 보여, 업그레이드는 어떨까 하는 생각입니다.

* Groovy 2.4
* Spring 4.1 and Spring Boot 1.2
* Gradle build system. (Gant는 Deprecated 됐습니다.)
* GGTS 및 IntelliJ Community Edition 지원이 나아졌습니다.

다른 중요한 이슈들도 있지만, 이 쪽도 한번 생각해 볼 만하여 이슈로 생성합니다~

**Comment by 1angerhans** [comment]
헐 하루만에 버전업됬네요. 
저도 같은 생각입니다. 기존 스프링 사용자에게 훨씬 친근할 것 같아서 이기회에 3.0으로 업그레이드하고 Groovy & Grails에 관한 연재를 해볼까 생각중입니다.
다만 지금 플러그인 의존도가 높아서 조만간 여유가되면 파일럿을 제대로 띄워봐 야될꺼 같아요. 겉어낼 플러그인도 좀 선별하구요.
조만간 브런치로 올릴께요. 도움 주시면 감사드리겠습니다 __)

**Comment by herblover** [comment]
흐음... 좀 살펴 봤더니 생각보다 작업량이 많아 보이네요.
일단 Plugin Repository도 변경되었고... 공식 문서에서 3.X 로 업그레이드 시에는 새 프로젝트를 3.X 기반으로 만들고 한땀한땀(;;;;) 손으로 복사하면서 테스트 하는걸 권장하네요.
이거 보통 작업이 아니겠는데요. ㅎㅎ
일단 할 수 있는 곳 까지 해 보죠 뭐. ^^;

**Comment by herblover** [comment]
헐... 새로 프로젝트 하나 만들어서 작업해 보고 있는데...
이거 보통 일이 아니네요. ;;;;
일단 기본 컨테이너가 Tomcat 으로 바뀌어서, build 쪽에 tomcat dependency는 필요 없어진거 딱 하나 좋고...
markdown, recaptcha, spring-security-*, twitter-bootstrap 등등 플러그인이 없네요... ;;;
spring-security-* 시리즈는 spring boot 을 통해서 처리하라는거 보니... 빠른 시간에 마이그레이션 하긴 어려울 듯 합니다. ;;;;

**Comment by 1angerhans** [comment]
네 저도 잠깐 해봤는데, 역시 플러긴쪽이 막막하네요 ㅠㅠ.. 예제보니 플러긴 소스를 직접 넣어주는 식으로 작업하던데요. 우선 시큐리티쪽을 그렇게 해서 올려봐야겠요.

**Comment by aragorn** [comment]
[Upgrading From 2.4.x To Grails 3.1](https://github.com/aragorn/grails-exploration/wiki/Upgrading-From-2.4.x-To-Grails-3.1)
아직 업그레이드 작업을 완료하지 못한 상태이긴 한데, build error 까지는 해결한 상태의 경험을 공유합니다. 도움이 되면 좋겠네요. ^^
